Client/Frontend/Home/RecentlySaved/RecentlySavedCellViewModel.swift

```diff
@@ -152,18 +95,13 @@ extension RecentlySavedCellViewModel: HomepageViewModelProtocol, FeatureFlaggabl
     }
 
     var hasData: Bool {
-        return !recentBookmarks.isEmpty || !readingListItems.isEmpty
+        return false
     }
```

Comment:
If `hasData` is always false, I'm not sure how the rest of the code will behave. `hasData` is used in `shouldShow` to know if we need to show the section. `shouldShow` is used in three places at the moment: 
- `updateEnabledSections()` in HomepageViewModel 
    - Since `shouldShow` is false, the reading list section will never be added to `shownSections`. `getPositionToInsert` wouldn't be good in that case. 
- `func createLayout() -> UICollectionViewLayout` in HomepageViewController
    - It seems the layout wouldn't be loaded if there's no data, since we check for `viewModel.shouldShow`
- `viewForSupplementaryElementOfKind` in HomepageViewController
    - Header configuration won't happen on this section it seems too

Reply:
I would think this would just need to be `return !recentItems.isEmpty` ?